docs/markdown/Commands.md

```diff
@@ -132,32 +132,50 @@ meson configure builddir -Doption=new_value
 *(since 0.54.0)*
 
 ```
-$ meson compile [-h] [-j JOBS] [-l LOAD_AVERAGE] [--clean] [-C BUILDDIR]
+$ meson compile [-h] [--clean] [-C BUILDDIR] [-j JOBS] [-l LOAD_AVERAGE]
+                [TARGET [TARGET ...]]
 ```
 
 Builds a default or a specified target of a configured meson project.
 
 ```
+positional arguments:
+  TARGET                                Targets to build. Target has the
+                                        following format: [PATH_TO_TARGET/]TARGE
+                                        T_NAME[:TARGET_TYPE]
```

Comment:
You should document that target is new in 0.55.0

Reply:
Fixed